examples/truncate-text-bento.amp.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
```

Comment:
Not sure if these are still necessary with Storybook now, but figured they couldn't hurt to include?

Reply:
It's probably fine.

Reply:
I gravitate to storybooks now in my preference. This doesn't hurt, of course.